dockers/docker-database/Dockerfile.j2

```diff
@@ -32,9 +32,21 @@ RUN sed -ri 's/^(save .*$)/# \1/g;                       \
              s/^logfile .*$/logfile ""/;                 \
              s/^# syslog-enabled no$/syslog-enabled no/; \
              s/^# unixsocket/unixsocket/;                \
+             s%pidfile .*%pidfile /var/run/redis/redis.pid%; \
```

Comment:
>pidfile /var/run/redis/redis.pid [](start = 26, length = 32)

Why changing?

Reply:
> > pidfile /var/run/redis/redis.pid [](start = 26, length = 32)
> 
> Why changing?

Here we want to use the same style when naming pid file, when we have mutiple DB instances, we are using instance name as pid, sock names. The default DB should use the same style, instead of hardcoded redis-server.pid, which is hard to apply the basename ${0} parameter.